### PR TITLE
Phone styles

### PR DESCRIPTION
**Pull request by ankitshah811**
TO DO:

* [x] @mehulkar Can you adjust the checkbox for phone numbers to be checked by default? I spent some time trying to figure out how to autocheck it, but everything I was doing didn't work. Current copy in this PR assumes it's checked
* [ ] Add phone numbers to host dashboard view for attendees who have shared number

### CODE REVIEW

File: app/views/tea_times/show.html.erb [hunk 1]
```diff
@@ -5,33 +5,59 @@
...
-            <% end %>
+            <div class="field phone split1 last">
+              <% if !current_user.phone_number.blank? %>
+                <div class="form-label">
+                  <%= f.label :phone_number, "Phone number", class: 'capitalize boldweight inline' %>
+                  <span class="sub-label light">
+                    We have your number down as <%= current_user.phone_number %>
+                  </span>
+                </div>
+                <%= f.check_box :provide_phone_number %>
```
Comment (ankitshah811) [inline app/views/tea_times/show.html.erb]: Couldn't figure out how to have this checked by default. If it's quick, could you handle this @mehulkar ?

Comment (mehulkar) [inline app/views/tea_times/show.html.erb]: done.

**Comment by mehulkar** [comment]
is this ready to merge? not sure if you are planning to add the phone numbers to host dashboard in this PR or in another one.

**Comment by ankitshah811** [comment]
@mehulkar would be done if we could get the conditions outlined in #622 sorted
if that will take more than a few min, let's deploy without it and get that done after

**Comment by mehulkar** [comment]
yeah can't do #622 in a few mins. ping me when you want me to deploy


------
Mehul Kar
@mehulkar

**Pull request merged**